nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/add", (req, res) => {
+
+    const first = Number(req.query.first);
+    const second = Number(req.query.second);
+    const addResult = first + second;
+    if (first && second) {
+        res.send(`add result is : ${addResult}`)
+    }
+    if (first) {
+        res.send(`you only type the firstNumber only ${first}`)
+    }
+    if (second) {
+
+        res.send(`you only type the secondNumber only${second}`)
+    }
+
+});
+
+app.get("/multiply/:first/:second", (req, res) => {
+    const first = Number(req.params.first);
+    const second = Number(req.params.second);
+    const multiplyResult = first * second;
+    if (first && second) {
+        res.send(`multiply result is : ${multiplyResult}`)
+    }
+    if (first) {
+        res.send(`you only type the firstNumber only ${first}`)
+    }
+    if (second) {
+
+        res.send(`you only type the secondNumber only${second}`)
+    }
```

Comment:
Did you test this out? What results do you get? I think you should check for the absence of first and second and send the messages so you'll have something like ``if(!first....){ do something}``. Let me know the outcome.

Reply:
if (addResult) {
        res.send(`add result is : ${addResult}`) 
    }else{
        res.send(`check the number you typed`)
    }
how do you think to judge it this way?

Reply:
haha, yes you are right I should write like the way you mentioned.